sdk/storage/perf-tests/storage-blob/package.json

```diff
@@ -12,7 +12,6 @@
     "@azure/storage-blob": "^12.5.0-beta.1",
     "@azure/test-utils-perfstress": "^1.0.0",
     "@types/uuid": "^8.0.0",
-    "@types/dotenv": "8.2.0",
```

Comment:
Why remove the "dotenv" dependency. storage-blob perf-test doesn't requires env now?

Reply:
It is not needed anymore.

`@types/dotenv` has been deprecated and simply importing dotenv would get the types.

https://www.npmjs.com/package/@types/dotenv